types/tx_result.go

```diff
@@ -0,0 +1,14 @@
+package types
+
+import (
+	abci "github.com/tendermint/abci/types"
+)
+
+// TxResult contains results of executing the transaction.
+//
+// One usage is indexing transaction results.
+type TxResult struct {
+	Height    uint64                 `json:"height"`
```

Comment:
right. I feel like we did discuss before but I didn't remember conclusion. We use `int` all over the codebase where we could/should use `uint64` so I wonder if we should break with that or at some point just fix it everywhere. So what about the `uint32` for the index? 

@jaekwon